nucypher/blockchain/eth/sol/source/contracts/proxy/Dispatcher.sol

```diff
@@ -39,6 +48,22 @@ contract Dispatcher is Upgradeable {
         emit Upgraded(address(0), _target, msg.sender);
     }
 
+    //------------------------ERC897------------------------
+    /**
+     * @notice ERC897, whether it is a forwarding (1) or an upgradeable (2) proxy
+     */
+    function proxyType() external pure override returns (uint256) {
```

Comment:
Small question - why `external` and not `public` here, just because we dont use it internally anywhere? (Comparing with the standard here: Interface here https://github.com/ethereum/EIPs/pull/897)

Reply:
yes, also interface can't have `public`, only `external` (after some version of solidity)